src/main/java/Assignment14.java

```diff
@@ -0,0 +1,18 @@
+class Actor{
+    public void work(){
+        System.out.println("Doing Acting");
+    }
+}
+class Director {
+    public void work(){
+        System.out.println("Directing");
+    }
+}
+public class Assignment14 {
+    public static void main(String[]args){
+        Actor act = new Actor();
+        Director dir = new Director();
+        act.work();
+        dir.work();
```

Comment:
you can direclty write like this too
new Actor().work();
new Director().work();